components/dashboard/src/components/ContextMenu.tsx

```diff
@@ -185,3 +165,47 @@ function ContextMenu(props: ContextMenuProps) {
 }
 
 export default ContextMenu;
+
+type MenuEntryProps = ContextMenuEntry & {
+    isFirst: boolean;
+    isLast: boolean;
+};
+export const MenuEntry: FunctionComponent<MenuEntryProps> = ({
+    title,
+    href,
+    link,
+    active,
+    separator,
+    customContent,
+    customFontStyle,
+    isFirst,
+    isLast,
+    onClick,
+}) => {
+    const clickable = href || link || onClick;
+
+    return (
+        <div
+            title={title}
+            className={cn(
+                "px-4 flex leading-1 py-3 text-sm",
```

Comment:
**suggestion**: Not sure if this will affect other elements, but the sub-pages could look slightly better with a smaller height and maybe even less font weight.

```suggestion
                "px-4 flex leading-1 py-2 text-sm",
```

Reply:
Agreed - I'm adding a `tight` option that uses the smaller y padding so that all of our other context menus don't tighten up as well. I do kinda like the tighter context menus in general though, but will leave that as a separate decision if we wanted to reduce the y padding on the items in there across the board.

Reply:
Maybe I'll go ahead and drop the `tight` prop option, and just swap the context menus to use the smaller y padding then if you're ok with it (would rather not have the extra prop for variance if we're ok w/ slimming them down slightly).